### ISSUE TITLE
`init` binding doesn't work in conjunction with `checked` binding

### ISSUE THREAD

**Issue by jabberwik** [ErikSchierboom/knockout-pre-rendered#19]
For example, with element:
```html
<input name="foo" type="checkbox" checked="checked" data-bind="init, checked: foo" />
```
The console error is
> Unable to process binding "init: function (){return undefined }"
> Message: fieldAccessor is not a function

I can work around this by using `{ field: foo, value: true }` but it's a little clunky. Since `checked` is a built-in binding, it would be cool if the two worked together.

**Comment by ErikSchierboom** [comment]
I'm a bit busy on the moment, but I'll look into it!

**Comment by ErikSchierboom** [comment]
@jabberwik This has been fixed in version 0.5.0.
